solr/contrib/ltr/src/java/org/apache/solr/ltr/util/LTRRerankHelper.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.ltr.util;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Weight;
+import org.apache.solr.common.SolrException;
+import org.apache.solr.common.SolrException.ErrorCode;
+import org.apache.solr.common.params.SolrParams;
+import org.apache.solr.ltr.feature.LTRScoringAlgorithm;
+import org.apache.solr.ltr.ranking.ModelQuery;
+import org.apache.solr.ltr.ranking.ModelQuery.ModelWeight;
+import org.apache.solr.ltr.rest.ManagedFeatureStore;
+import org.apache.solr.ltr.rest.ManagedModelStore;
+import org.apache.solr.request.SolrQueryRequest;
+
+public class LTRRerankHelper {
+  private final SolrQueryRequest request;
+  private final SolrParams params;
+  
+  public LTRRerankHelper(SolrQueryRequest request, SolrParams params){
+    this.request = request;
+    this.params = params;
+  }
+  
+  public ManagedModelStore getModelStore(){
+    return (ManagedModelStore) request.getCore().getRestManager()
+        .getManagedResource(CommonLTRParams.MODEL_STORE_END_POINT);
+  }
+  
+  public ManagedFeatureStore getFeatureStore(){
+    return (ManagedFeatureStore) request.getCore().getRestManager()
+        .getManagedResource(CommonLTRParams.FEATURE_STORE_END_POINT);
+  }
+  
+  
+  public LTRScoringAlgorithm getModel(String modelName){
+    ManagedModelStore modelStore = getModelStore();
+    final LTRScoringAlgorithm rerankModel = modelStore.getModel(modelName);
+    if (rerankModel == null) {
+      throw new SolrException(ErrorCode.BAD_REQUEST,
+          "cannot find " + CommonLTRParams.MODEL + " " + modelName);
+    }
+    return rerankModel;
+  }
+  
+  public String getDefaultFeatureStoreName(){
```

Comment:
We can remove this function, callers can use the variable directly without a function to wrap it